<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeesView</class>
 <widget class="QDialog" name="EmployeesView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Employees</string>
  </property>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QTableView" name="employeesTableView">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>1040</width>
     <height>330</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tasksTableView">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>370</y>
     <width>1040</width>
     <height>330</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
